elements/pfelement/src/pfelement.scss

```diff
@@ -12,15 +12,20 @@ body {
 
 body[unresolved] {
   opacity: 0;
+  visibility: hidden;
```

Comment:
@mwcz - I don't think you meant to include the CSS changes in this PR. Can you pull these out?

Reply:
Whoops, thanks, I'll pull those out.

Reply:
After merging master, this doesn't matter any more.